mac/mac.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "libmac",
+  "no-autogen": true,
+  "rm-configure": true,
+  "make-install-args": [
+    "prefix=/app"
+  ],
```

Comment:
Surely this is missing all the `"cleanup"` for left over headers and whatnot.

Reply:
Alright, 67f6320 adds the `cleanup` for headers. The `install` target installs only `/bin/mac`, `/lib/libMAC.so` and `/include/MAC/`.

Reply:
Good for me. Works in the test I have

Reply:
Is the binary `mac` used often?

Reply:
- ca.littlesvr.asunder
- com.github.Flacon

Both need it. Not sure if that counts as much, but otherwise it would kind of invalidate all this work. The whole point behind making this shared-modules, is so I and enzo can share some work on some very finicky codecs.

Reply:
Just checking. Thanks!